docs/go.mod

```diff
@@ -10,3 +10,5 @@ require (
 	github.com/spf13/cobra v1.1.3
 	gopkg.in/yaml.v2 v2.4.0
 )
+
+replace github.com/qri-io/qri => ../
```

Comment:
Actually would prefer not to, as it does not work otherwise and is a bit hard to keep in sync. We would have to always first commit the changes then follow up with a version bump to it otherwise.

This way it always runs against "current qri core" whatever that might be, which is also what it should test.